optuna/cli.py

```diff
@@ -343,6 +343,7 @@ class _Studies(_BaseCommand):
 
     _study_list_header = [
         ("name", ""),
+        ("user_attrs", ""),
```

Comment:
According to https://github.com/optuna/optuna/issues/3899#issuecomment-1232431835, the purpose of this PR is to make `optuna studies` and `optuna trial` consistent in terms of `user_attrs`.
On the other hand, current implementation is inconsistent when the `user_attrs` are empty.

*optuna trial*
`user_attrs` column is hidden when no trials have `user_attrs`.

```python
import optuna

study = optuna.create_study(
    storage="sqlite:///4129.db",
    study_name="4129",
)
```

```console
% study.optimize(objective, n_trials=2)
+--------+-------+---------------------+---------------------+----------------+----------+
| number | value | datetime_start      | datetime_complete   | duration       | state    |
+--------+-------+---------------------+---------------------+----------------+----------+
|      0 |   1.0 | 2022-11-06 14:25:54 | 2022-11-06 14:25:54 | 0:00:00.003276 | COMPLETE |
+--------+-------+---------------------+---------------------+----------------+----------+
```

*optuna studies*
`user_attrs` is always shown.

```console
% optuna create-study --storage sqlite:///4129-empty.db --study-name empty 
%  optuna studies --storage sqlite:///4129-empty.db                                                                                                                   [~/pfn/code/optuna]
+-------+------------+---------------+----------+----------------+
| name  | user_attrs | direction     | n_trials | datetime_start |
+-------+------------+---------------+----------+----------------+
| empty | {}         | ('MINIMIZE',) |        0 |                |
+-------+------------+---------------+----------+----------------+
```

Do we align the behavior when the `user_attrs` values are empty?
@g-votte @nzw0301

Reply:
As @toshihikoyanase mentions, the goal is to make `optuna studies` and `optuna trials` consistent. It is the case even for the edge case where `user_attrs` is empty.

Reply:
True I will fix this